Makefile

```diff
@@ -0,0 +1,8 @@
+.PHONY: test
+test:
+	go test -tags=assert -race ./...
```

Comment:
`-tags=assert` is for Arrow tests, we don't need this I think... just fyi :) (still it could be beneficial to keep it...)

Reply:
Ah, I wasn't quite sure what that was for. Good to know.

Reply:
Yeah good call, let's remove it :)

Reply:
Sorry, missed the request - follow up [here](https://github.com/cloudquery/cloudquery-api-go/pull/41)